internal/internal_test.go

```diff
@@ -913,6 +967,14 @@ func testAuthzServerWithObjectDecision(customLogger plugins.Plugin, dryRun bool)
 	return s
 }
 
+func createCheckRequest(module string) *ext_authz.CheckRequest {
```

Comment:
Nit: I'd rename this parameter to reflect it's purpose (looks like it was copy-pasted from below?)

Reply:
Done